custom/icds_reports/views.py

```diff
@@ -642,6 +668,19 @@ def post(self, request, *args, **kwargs):
                 loc_level=aggregation_level,
                 show_test=include_test
             ).to_export('csv', location)
+        elif indicator == 7:
```

Comment:
probably not worth addressing wholesale, but just noting more magic numbers here